tickit/devices/eiger/eiger_settings.py

```diff
@@ -0,0 +1,94 @@
+from dataclasses import dataclass, field
+from typing import List
+
+from .eiger_schema import (
+    FLOAT_GRID,
+    UINT_GRID,
+    field_config,
+    ro_float,
+    ro_str,
+    rw_bool,
+    rw_float,
+    rw_int,
+    rw_str,
+)
+
+FRAME_WIDTH: int = 4148
+FRAME_HEIGHT: int = 4362
+
+
+# TODO: Figure out how to properly declare the 2d arrays for flatfield and pixel_mask,
+# as they return 'None' when requesting.
```

Comment:
Is this TODO still relevant? They look okay to me. Or are you trying to declare them as numpy arrays rather than lists?

Reply:
Ah yeah this was fixed a a few commits back but i forgot to remove it :sweat_smile: